src/components/ConfigDetailsChanges/ConfigDetailsChanges.tsx

```diff
@@ -39,85 +76,136 @@ export function ConfigDetailsChanges({
   }
 
   return (
-    <div className="overflow-hidden bg-white">
+    <div
+      className="overflow-hidden bg-white cursor-pointer"
+      onClick={(e) => {
+        setOpen(true);
+      }}
+    >
+      <Modal
+        title={
+          <>
+            <Icon
+              name={changeDetails?.change_type}
+              secondary="diff"
+              className="w-5 h-auto pr-1"
+            />
+            {changeDetails?.change_type}
+          </>
+        }
+        open={open}
+        onClose={(e) => {
```

Comment:
Why do we need this ?

Reply:
When a Modal is opened on top of another Modal close is not working properly. In order to fix that issue I have added this.

Reply:
But then shouldn't it be on all modals ? and/or add a comment on why it is needed

Reply:
I'll add a comment